app/controllers/agenda-items.ts

```diff
@@ -9,12 +9,15 @@ import { ModelFrom } from 'frontend-burgernabije-besluitendatabank/lib/type-util
 import AgendaItemsRoute from 'frontend-burgernabije-besluitendatabank/routes/agenda-items';
 import KeywordStoreService from 'frontend-burgernabije-besluitendatabank/services/keyword-store';
 import MunicipalityListService from 'frontend-burgernabije-besluitendatabank/services/municipality-list';
+import AgendaItem from 'frontend-burgernabije-besluitendatabank/models/agenda-item';
 
 export default class AgendaItemsController extends Controller {
   @service declare router: RouterService;
   @service declare store: Store;
   @service declare keywordStore: KeywordStoreService;
   @service declare municipalityList: MunicipalityListService;
+
+  @tracked agendaItems: AgendaItem[] = [];
```

Comment:
Not sure if this is the correct type, but it acts like it so good enough for now I guess.